main/parse.c

```diff
@@ -3890,15 +3890,21 @@ extern bool runParserInNarrowedInputStream (const langType language,
 
 static bool createTagsWithFallback (
 	const char *const fileName, const langType language,
-	MIO *mio)
+	MIO *mio, bool *failureInOpenning)
 {
 	langType exclusive_subparser = LANG_IGNORE;
 	bool tagFileResized = false;
 
 	Assert (0 <= language  &&  language < (int) LanguageCount);
 
 	if (!openInputFile (fileName, language, mio))
+	{
+		if (failureInOpenning)
+			*failureInOpenning = true;
 		return false;
+	}
+	if (*failureInOpenning)
```

Comment:
Shouldn't this be 'if (failureInOpenning)'  instead of 'if (*fail...)' ?

Reply:
You are correct. Thank you very much!